components/brave_ads/core/internal/creatives/creative_ads_database_table.cc

```diff
@@ -85,8 +88,10 @@ CreativeAdInfo FromMojomRow(const mojom::DBRowInfoPtr& mojom_db_row) {
   creative_ad.per_month = ColumnInt(mojom_db_row, 4);
   creative_ad.total_max = ColumnInt(mojom_db_row, 5);
   creative_ad.value = ColumnDouble(mojom_db_row, 6);
-  creative_ad.split_test_group = ColumnString(mojom_db_row, 13);
-  creative_ad.target_url = GURL(ColumnString(mojom_db_row, 7));
+  creative_ad.split_test_group = ColumnString(mojom_db_row, 7);
```

Comment:
why column number is changed from 13 to 7?

Reply:
This was a bug :-). However, did not break anything because are not using split test groups at this time.